docs/operations/durable-storage.md

```diff
@@ -0,0 +1,47 @@
+---
+id: durable-storage
+title: "Durable storage for the multi-stage query engine"
+sidebar_label: "Durable storage"
+---
+
+You can use durable storage to improve querying from deep storage and SQL-based ingestion.
+
+> Note that only S3 is supported as a durable storage location.
+
+Durable storage for queries from deep storage provides a location where you can write the results of deep storage queries to. Durable storage for SQL-based ingestion is used to temporarily house intermediate files, which can improve reliability.
+
+## Enable durable storage
+
+To enable durable storage, you need to set the following common service properties:
+
+```
```

Comment:
@cryptoe are there any new durable storage configs specifically for query from deep storage? Or are results written to the `tempDir` property?

Reply:
The tempDir is used more as a staging directory before pushing out bytes to s3. Its not related to results of the query.